src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -482,9 +498,9 @@ COMMENT ON COLUMN defendant.cas_id
 IS 'foreign key from court_case';
 
 CREATE TABLE device_register

```

Comment:
Looking at @karen-hedges PR from earlier - she was expecting the table to be named `node_register`?
https://github.com/hmcts/darts-api/pull/359/files#diff-dfb5b8a34cd01a3fe9a384a7def51f31c027fb5f51c0e3b7c77143f2c219e828R5